ptcg_cockatrice.xml

```diff
@@ -175,6 +266,17 @@
 			<text>1B: Evolve to Nidorina
 				2B: Search your library for a card named Nidoran♀ or Nidoran♂ and put it onto the battlefield. Then shuffle.</text>
 		</card>
+		<card>
+			<name>Nidoqueen</name>
+			<set picURL="https://images.pokemontcg.io/ecard2/96_hires.png" rarity="uncommon">PTCG</set>
+			<color>B</color>
+			<manacost>6B</manacost>
+			<type>Pokémon - Poison Ground</type>
+			<pt>4/7</pt>
+			<tablerow>2</tablerow>
+			<text>3BB: Evolve from Nidorina
+				If a Pokémon you control with no +1/+1 counters on it would be put into the graveyard from the battlefield, exile it instead. Return it to the battlefield under its owner's control with a +1/+1 counter on it at the beginning of the next end step.</text>
```

Comment:
```suggestion
				If a Pokémon you control with no counters on it would be put into a graveyard from the battlefield, you may exile it instead. If you do, return it to the battlefield under its owner's control at the beginning of the next end step. Put a +1/+1 counter on it.</text>
```